lms/djangoapps/certificates/migrations/0011_certificategenerationcoursesetting_can_self_generate.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import apps
+from django.db import migrations, models
+from django.db.models import F
+
+def copy_field(apps, schema_editor):
+    MyModel = apps.get_model('certificates', 'certificategenerationcoursesetting')
+    MyModel.objects.all().update(can_self_generate=F('enabled'))
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('certificates', '0010_certificate_language_course_setting'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='certificategenerationcoursesetting',
+            name='can_self_generate',
+            field=models.BooleanField(default=False),
```

Comment:
Depending on what we decide regarding how this migration should be run, we might not want to set a default for the new field. I can see it being valuable to be able to tell whether the field has been explicitly set to something (True/False), or if it has yet to be set at all (Null).